aws/track.yml

```diff
@@ -0,0 +1,77 @@
+slug: aws-cloud-account
+id: suhbhzjy3lxo
+type: track
+title: AWS Cloud Account
+teaser: Learn how to build tracks with an AWS account
+description: |-
+  Learn how to use a sandboxed AWS account in your Instruqt track.
+
+  **This track demonstrates how to:**
+    - Link an AWS cloud account to your sandbox
+    - How to use the cloud client Docker container
+    - Use the AWS CLI and UI from our learn environment
+    - Limit usage of the cloud account using IAM and SCP policies
+icon: https://cdn.instruqt.com/assets/templates/aws.png
+tags:
+- Cloud Account
+- Docker
+owner: templates
+developers:
+- arslan@instruqt.com
+- matthisk@instruqt.com
+private: false
+published: true
+challenges:
+- slug: aws-ec2-create
+  id: stswhtvyegcl
+  type: challenge
+  title: AWS EC2 instance creation
+  teaser: Every cloud starts from VM
+  notes:
+  - type: text
+    contents: |-
+      You're about to create an EC2 instance using AWS CLI.
+
+      Please wait while we provision the AWS account.
+  assignment: |-
+    Use the terminal to create ec2 instance:
+
+    ```
+    aws ec2 run-instances --image-id ami-01685d240b8fbbfeb --instance-type t2.nano
+    ```
+
+    To complete this challenge, press **Check**.
+  tabs:
+  - title: Cloud CLI
+    type: terminal
+    hostname: cloud-client
+  - title: AWS Console
+    type: service
+    hostname: cloud-client
+    path: /
+    port: 80
+  difficulty: basic
+  timelimit: 600
+- slug: aws-ec2-delete
+  id: sxil4smudtyw
+  type: challenge
+  title: AWS EC2 instance deletion
+  teaser: Use AWS console to manage your EC2 virtual machine.
+  notes:
+  - type: text
+    contents: You just created your EC2 virtual machine, let's try to remove it!
+  assignment: |-
+    Login to AWS console using credentials displayed in AWS Console tab, navigate to Services -> EC2 -> Instances,
+    and terminate instance you created.
+
+    To complete this challenge, press **Check**.
```

Comment:
Is there a reason we're not doing this in the CLI? Is it to show off the ability to give access to the aws console through the AWS IAM tab?

Perhaps give both options, so you could also shut it down with `terminate-instances`

WDYT?